pkgs/applications/networking/sync/rsync/rrsync.nix

```
@@ -1,11 +1,13 @@
-{ lib, stdenv, fetchurl, perl, rsync, fetchpatch }:
+{ lib, stdenv, fetchurl, python3, rsync, fetchpatch }:
```

If you feel like it, you can clean these up. I think `lib`, `fetchurl` and `fetchpatch` are unused.

---

```suggestion
{ stdenv, python3, rsync }:
```
I've tested this, you're correct. It yields the same derivation.
@ifurther Feel free to apply that suggestion :)